Documentation/bpf.rst

```diff
@@ -2075,9 +2075,10 @@ describe some of the differences for the BPF model:
 10. **Remove struct padding with aligning members by using #pragma pack.**
 
   In modern compilers, data structures are aligned by default to access memory
-  efficiently. Structure members are aligned to memory address that multiples their
-  size, and padding is added for the proper alignment. Because of this, the size
-  of struct may often grow larger than expected.
+  efficiently. Structure members are packed to memory addresses that fit their
+  size, and padding is added for the proper alignment with the processor word size
+  (e.g. 8-byte for 64-bit processors, 4-bytes for 32-bit processors).
+  Because of this, the size of struct may often grow larger than expected.
```

Comment:
The first part of this sentence sounds odd to me, although I understand what it tries to say

> Structure members are aligned to memory address that multiples their size

I rephrased it, but let me know if you think of something better

Reply:
The new "fit their size" wording seems a bit ambiguous to me, but I'm not an expert in alignment. My understanding is that the size of the members doesn't matter so much other than how that size relates to the processor word size, in which case the latter part of the new sentence seems to pull the most weight in this piece. Maybe just drop the "fit their size" bit?

I think I'll tag-team @kaworu in here, he may have thoughts on how to accurately describe this piece.